v2/controllers/marketplace/razeedeployment_controller.go

```diff
@@ -200,6 +217,26 @@ func (r *RazeeDeploymentReconciler) SetupWithManager(mgr manager.Manager) error
 		Complete(r)
 }
 
+// +kubebuilder:rbac:groups="",resources=configmaps;pods;secrets,verbs=get;list;watch
```

Comment:
is there a way to request those permissions only if Razee is enabled?
`instance.Spec.Enabled`

Reply:
Unfortunately no. Permissions ship with the operator. To change permissions requires permissions to modify the roles.

Reply:
is it fair to say that the global access to secrets, namespaces and nodes is mostly due to Razee?

Reply:
Yes. That level of permission is mostly related to Razee. There are some cases where wider permissions are required but can be scoped down to resources or where I can resource names. 

I think I can scope down the gets as well. Global secret get is probably not required. When we merge these stories I'll take a hard look at everything again.

Reply:
Right. Global get for all secrets is what needs to be narrowed. Thx!